rollup.config.js

```diff
@@ -14,7 +14,7 @@ export default {
         {dest: 'dist/glamorous.es.js', format: 'es'},
         {dest: 'dist/glamorous.cjs.js', format: 'cjs'},
   ],
-  exports: 'default',
+  exports: 'named',
```

Comment:
We may have to utilize that commented out code to get what we want. I've added a `dist-test` to make sure we don't break the way we currently export the module and when I changed the `exports` in the rollup config to `named` it breaks the `require` API (you have to do `require('glamorous').default` which I really want to avoid).

That commented out code [was an accident](https://medium.com/@kentcdodds/please-don-t-commit-commented-out-code-53d0b5b26d5f), but it came from [match-sorter](https://github.com/kentcdodds/match-sorter) to overcome limitations noted in [the rollup docs for `exports`](https://github.com/rollup/rollup/wiki/JavaScript-API#exports).

I'll probably merge the `dist-tests` here in a minute. If you could rebase onto that and try to make it work that'd be great. If you need a hand, let me know!